### DISCUSSION TITLE
多卡运行为什么会报错？No CUDA GPUs are available

### DISCUSSION THREAD

**Discussion by GzRichChen** [zai-org/ChatGLM3#623] [Q&A] [answered]
model = load_model_on_gpus(model_path, num_gpus=4)),报错：RuntimeError: No CUDA GPUs are available，但是确实是有4张卡，用chatglm2也可以用多卡，为什么chatglm3会报错

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
更新代码
